package.json

```diff
@@ -326,7 +332,8 @@
       "github:assemblyscript/assemblyscript": false,
       "tiny-secp256k1": false,
       "@lavamoat/preinstall-always-fail": false,
-      "fsevents": false
+      "fsevents": false,
+      "metamask-crx": false
```

Comment:
This was necessitated by the `postinstall` script I guess? :thinking: Not sure we need that script, since it won't run anyway.